articles/quickstart/spa/_includes/_authz_preamble.md

```diff
@@ -1,16 +1,16 @@
-The previous step covers how to protect the resources served by your API such that only users who have authenticated in your application (and who are properly authorized) can access them. While this deals with protecting data resources from your server, it is likely that you will also need some way to control authorization in your client-side single page application.
+This tutorial shows you how to control authorization in your client-side single page application.
 
 ## Access Control in Single Page Apps
 
-Distinguishing between different users and controlling what they can and cannot access is typically known as access control. The way that access control is implemented depends on the needs of your application, but with Auth0 the most common approach is to use the `scope`s granted to a user as a way to make decisions about what they can see in the application and which routes they have access to.
+Access control is the control over what different users can see in your application and which routes they can access.
+With Auth0, you can implement access control by using scopes granted to users.
 
-For single page applications which get their data from APIs, there are two major things that need to be considered when authorization and access control are introduced:
+To set up access control in your application, define the following restrictions:
+* The data from an API can only be released if the user is authorized to access it. 
+* The user can access specific routes and UI elements in your application only if they have the appropriate access level.
 
-1. The particular data being requested from the API should only be released if the user making the request is authorized to access it
-2. The user should only be able to access client-side routes and see certain UI elements if they have an appropriate access level to do so
-
-The previous step used a `scope` of `read:messages` for access to API resources. Since this `scope` indicates that the user has read-only access to data, it might be considered that the user has some kind of "regular user" access level. If you wanted some users to have write access to the same resource, and therefore some kind of "administrator" access level, you might consider introducing a `scope` of `write:messages`.
+The previous step used the `read:messages` scope for accessing API resources. This scope indicates that the user can only view the data. You can consider users with this scope as regular users. If you want to give some users permission to edit the data, you can add the `write:messages` scope. 
 
 ::: note
-Auth0 makes no assumptions about how the `scope`s for your API map to various access levels, nor is there any restriction on what you call your `scope`s and access levels. These details are at your discretion.
-:::
+You can give your scopes any names and map them to any access levels you want. 
```

Comment:
No you cannot give any names you want. You cannot use standard claims names, and you need to namespace the custom ones 

https://auth0.com/docs/scopes

https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims

Reply:
How about omitting the rather problematic 'any name you want' and saying:
Read about naming scopes and mapping them to access levels in the [Scopes documentation](/scopes). 

Since there are some rules to follow.